package.json

```diff
@@ -1,47 +1,49 @@
 {
-  "name": "@signalk/client",
-  "version": "0.2.1",
-  "description": "Signal K JavaScript Client",
-  "main": "./dist/index.js",
+  "name": "@signalk/signalk-js-sdk",
+  "version": "1.0.0",
+  "description": "A Javascript SDK for Signal K servers. Provides various abstract interfaces for discovering (via optional mDNS) the Signal K server and communication via WebSocket & REST. Aims to implement all major APIs in the most recent Signal K version(s)",
+  "main": "index.js",
   "scripts": {
-    "test": "mocha",
-    "lint": "eslint index.js",
-    "prettier": "prettier --print-width 119 --single-quote --trailing-comma es5 --no-bracket-spacing --write *.js",
-    "changelog": "github-changes -o signalk -r signalk-js-client -a --only-pulls --use-commit-body --data=pulls",
-    "build": "babel index.js --out-dir dist",
-    "prepublish": "npm run lint && NODE_ENV=production npm run build",
-    "doc": "jsdoc -R README.md index.js"
+    "test": "mocha --compilers js:babel-register",
+    "start": "nodemon --exec babel-node src/index.js",
+    "dist": "./node_modules/.bin/babel src -d dist",
+    "prepublish": "./node_modules/.bin/babel src -d dist",
+    "postinstall": "./node_modules/.bin/babel src -d dist",
+    "push": "npm publish --access public --scope @signalk ."
   },
-  "keywords": [
-    "Signal K",
-    "SignalK",
-    "JavaScript"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/signalk/signalk-js-client"
-  },
-  "author": "Teppo Kurki <teppo.kurki@iki.fi>",
+  "keywords": [],
+  "author": "Fabian Tollenaar <fabian@decipher.industries> (http://signalk.org)",
   "license": "Apache-2.0",
-  "dependencies": {
-    "bluebird": "^3.5.0",
-    "debug": "^3.0.0",
-    "eventemitter3": "^3.0.0",
-    "lodash": "^4.0.1",
-    "superagent": "^3.5.2",
-    "superagent-promise": "^1.0.3",
-    "url": "^0.11.0",
-    "ws": "^3.0.0"
-  },
   "devDependencies": {
+    "casper-chai": "^0.3.0",
+    "chai": "^4.1.0",
+    "mdns": "^2.3.4",
+    "mocha": "^3.4.2"
+  },
+  "standard": {
+    "globals": [
+      "describe",
+      "before",
+      "after",
+      "it",
+      "expect",
+      "Promise",
+      "WebSocket"
+    ]
+  },
+  "dependencies": {
     "babel-cli": "^6.24.1",
-    "babel-preset-babili": "^0.1.2",
-    "babel-preset-es2015": "^6.24.1",
-    "chai": "^4.0.1",
-    "eslint": "^4.1.0",
-    "eslint-plugin-es5": "^1.1.0",
-    "github-changes": "^1.1.0",
-    "jsdoc": "^3.4.0",
-    "prettier": "^1.4.1"
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-register": "^6.24.1",
+    "bluebird": "^3.5.1",
+    "cross-fetch": "^2.1.1",
+    "debug": "^4.1.1",
+    "eventemitter3": "^3.0.1",
+    "isomorphic-ws": "^4.0.0",
+    "moment": "^2.20.1",
```

Comment:
Not used at all and everybody and their friend complains about it's size.

Reply:
Good point, I haven't bothered to check my dependencies yet.